### PR TITLE
Some safety around ACL allows.

### PR DESCRIPTION
**Pull request by Gnuxie**
Related: https://github.com/Gnuxie/Draupnir/issues/56.

Needs some kind of testing before merging.

### CODE REVIEW

File: src/models/ServerAcl.ts [hunk 1]
```diff
@@ -104,6 +104,9 @@ export class ServerAcl {
         if (!allowed || allowed.length === 0) {
```
Comment (Gnuxie) [inline src/models/ServerAcl.ts:104]: Wait a minute, `!allowed`?? What's that supposed to mean? And how are we pushing to it if its undefined??

Comment (TheArcaneBrony) [reply src/models/ServerAcl.ts:104]: Probably should make this `allowed = [ '*' ];`

Comment (Gnuxie) [reply src/models/ServerAcl.ts:104]: Why?

As for why `!allowed` was there, seemingly it was a redundant check from a long time ago

**Comment by Gnuxie** [comment]
This was incorporated into MPS

**Pull request closed without merging**